.yarnrc

```diff
@@ -0,0 +1 @@
+--install.ignore-engines true
```

Comment:
> To mitigate this I added yarnrc and changed script to run on linting only on latest lts.

Shouldn’t only the latter be enough?

Reply:
Yarn would fail to install if node version is not satisfied in one of the packages(typescript-eslint). Even if we run eslint v12 we still have problem with install.